pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>angularjs</artifactId>
-    <version>1.3.11-SNAPSHOT</version>
+    <version>1.3.12-SNAPSHOT</version>
```

Comment:
For `1.3.11` this needs to be `1.3.11-SNAPSHOT` since Maven does the actual release.